doc/documents/Examples_Tutorials/Examples_Tutorials.rst

```diff
@@ -0,0 +1,623 @@
+   +---------------------------+---------------------------------------------------------------+
+   |                           | .. code:: c++                                                 |
+   |                           |                                                               |   
+   |                           |    mli_status mli_krn_maxpool_chw_fx8_k3x3(                   |
+   |                           |       const mli_tensor * in,     // Input tensor              |
+   |                           |       const mli_pool_cfg * cfg,  // Pooling configuration     |
+   |                           |       mli_tensor * out           // Output tensor             |
+   |     **MLI Function**      |    );                                                         |
+   |                           |    mli_status mli_krn_avepool_chw_fx8_k3x3(                   |
+   |                           |       const mli_tensor * in,     // Input tensor              |
+   |                           |       const mli_pool_cfg * cfg,  // Pooling configuration     |
+   |                           |       mli_tensor * out           // Output tensor             |
+   |                           |    );                                                         |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+   
+   |                           | .. code:: c++                                                 |
+   |                           |                                                               |   
+   |                           |     mli_pool_cfg shared_pool_cfg = {                          |
+   |                           |        .kernel_height = 3, .kernel_width = 3,                 |
+   | **MLI Function Config**   |        .stride_height = 2, .stride_width = 2,                 |
+   |                           |        .padding_bottom = 1, .padding_top = 0,                 |
+   |                           |        .padding_left = 0, .padding_right = 1                  |
+   |                           |     };                                                        |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+   
+   
+..
+
+.. note:: 
+
+   All specializations for pooling and convolution group of primitives are declared in the appropriate header files (see it in the GitHub repository). Functions without specialization postfix work like switchers, analyzing parameters and choosing proper one to delegate actual job. This can be used in debug to define a proper specialization.    
+..
+
+Consider the last two operations:
+ 
+.. image::  ../images/6_op_map_step4.png
+   :align: center
+   :alt: CIFAR-10 Graph Visualization: Final Layers
+
+..
+
```

Comment:
Was "*Fully connected (referred as Inner Product in Caffe) and softmax don’t require any specific analysis*" part omitted intentionally here?